lib/default.nix

```diff
@@ -32,7 +44,7 @@ rec {
   createVolumesScript = pkgs: volumes:
     lib.optionalString (volumes != []) (
       lib.optionalString (lib.any (v: v.autoCreate) volumes) ''
-        PATH=$PATH:${with pkgs.buildPackages; lib.makeBinPath [ coreutils util-linux e2fsprogs xfsprogs dosfstools btrfs-progs ]}
+        PATH=$PATH:${with pkgs.buildPackages; lib.makeBinPath ([ coreutils ] ++ (collectFsUtils volumes pkgs))}
```

Comment:
I think it's no longer clear where `pkgs` comes from. We should get rid of the `with` clause.